app/boards/shields/helix/helix_right.conf

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2020 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+# Peripheral Unit (Right)
+CONFIG_ZMK_SPLIT=y
+CONFIG_ZMK_SPLIT_BLE_ROLE_PERIPHERAL=y
```

Comment:
```suggestion
```
This option has been removed recently in favor of a more concise way of defining splits. Remove this please :)

Reply:
I removed that line. If I need to edit the left config file, please let me know. Going to commit shortly.